db/seeds/post_flag_types.yml

```diff
@@ -32,3 +32,4 @@
   confidential: false
   active: true
   post_type_id: null
+

```

Comment:
This is just a newline at the end of the file. If you delete the newline then this file will disappear from the diff.

Reply:
Gone now, thanks.